lib/rubyhub/operations/configuration/setup.rb

```diff
@@ -5,25 +5,33 @@ module Operations
     module Configuration
       class Setup
         TEMPLATE_PATH = 'lib/rubyhub/config/templates/.rubyhub.yml'.freeze
+        DESCRIPTION_CONFIG_PATH = 'lib/rubyhub/config/templates/.description.txt'.freeze
         CONFIG_EXISTS_MESSAGE = 'Config already exists!'.freeze
         CONFIG_INSTALLED_MESSAGE = 'Config successfully installed!'.freeze
 
         class << self
           def call
-            return puts CONFIG_EXISTS_MESSAGE if Rubyhub::Configuration.exists?
+            initialize_template(TEMPLATE_PATH)
+            return if Rubyhub::Configuration.exists?(Rubyhub::Configuration::DESCRIPTION_CONFIG_PATH)
 
-            initialize_config
-            puts CONFIG_INSTALLED_MESSAGE
+            initialize_config(DESCRIPTION_CONFIG_PATH)
```

Comment:
why it called config? template is our config, isn't?

Reply:
In order to not populate our class with dozens of new methods and to not duplicate code, I was trying to re-use methods that are already persisted in our class.